homeassistant/components/vesync/const.py

```diff
@@ -59,4 +60,9 @@
     "LAP-V102S-AASR": "Vital100S",  # Alt ID Model Vital100S
     "LAP-V102S-WEU": "Vital100S",  # Alt ID Model Vital100S
     "LAP-V102S-WUK": "Vital100S",  # Alt ID Model Vital100S
+    "EverestAir": "EverestAir",
```

Comment:
Is this how the device is reported? If not let's use the model number without the variant attached to follow how the other models are done.

Reply:
@cdnninja, sorry, missed this earlier this week!  

My two EverestAirs report as LAP-EL551S-AUS.  I based the addition to const.py on the EverestAir models listed in [pyvesync's vesyncfan.py](https://github.com/webdjoe/pyvesync/blob/master/src/pyvesync/vesyncfan.py).  For the const.py file, I tried to match the existing style of the Core/Vital models (such as line 58), by listing "EverestAir": "EverestAir" first.  Happy to revise you'd prefer that not be there.

Reply:
Oops I missed this.  You are correct.